hollow/src/main/java/com/netflix/hollow/api/producer/metrics/AnnouncementMetrics.java

```diff
@@ -0,0 +1,64 @@
+package com.netflix.hollow.api.producer.metrics;
+
+import java.util.OptionalLong;
+
+public class AnnouncementMetrics {
+
+    private long dataSizeBytes;                                     // Heap footprint of announced blob in bytes
+    private long announcementDurationMillis;                        // Announcement duration in ms, only applicable to completed cycles (skipped cycles dont announce)
+    private boolean isAnnouncementSuccess;                          // true if announcement was successful, false if announcement failed
+    private OptionalLong lastAnnouncementSuccessTimeNanoOptional;   // monotonic time of last successful announcement (no relation to wall clock), N/A until first successful announcement
+
+
+    public long getDataSizeBytes() {
+        return dataSizeBytes;
+    }
+    public long getAnnouncementDurationMillis() {
+        return announcementDurationMillis;
+    }
+    public boolean getIsAnnouncementSuccess() {
+        return isAnnouncementSuccess;
+    }
+    public OptionalLong getLastAnnouncementSuccessTimeNanoOptional() {
```

Comment:
Minor point. Naming-wise i don't think it necessary to express the kind of return type in the method name. IMHO the return type already expresses that. In the case of the setter of the builder it looks a little odd :-)

Reply:
Done